### PR TITLE
Remove redux, switch to new service layer

### PR DESCRIPTION
**Pull request by davidpchi**

### CODE REVIEW

**Comment by davidpchi** [comment]
build failed, looking into it

File: src/playerOverview/PlayerScreen.tsx [hunk 1]
```diff
@@ -77,15 +76,19 @@ const processPlayerChampions = (
...
-    const championIdMap = useSelector(gameInfoSelector.getChampionMap);
+    const championIdMapResponse = DataDragonService.useChampionIdMap();
+    const championIdMap = championIdMapResponse.data ?? {};
 
     const playerClasses = useMemo(
         () => championClassWinRates(Object.values(player?.champions ?? {})),
         [player]
     );
 
+    console.log('wow');
```
Comment (michaeljvaldes) [inline src/playerOverview/PlayerScreen.tsx]: wow

**Review by michaeljvaldes — CHANGES_REQUESTED** [review]
wow

**Review by michaeljvaldes — APPROVED** [review]
YAS

**Pull request merged**